### PR TITLE
fix(zh-cn): deal with macro render problems

### PR DESCRIPTION
**Pull request by jasonren0403**
### Description

fix unrendered macro occurences

### Related issues and pull requests

Part of #13499

### CODE REVIEW

File: files/zh-cn/web/css/css_functions/index.md
Comment (jasonren0403) [inline files/zh-cn/web/css/css_functions/index.md:1]: This file needs a complete retranslate, the content seems to have rewritten.

File: files/zh-cn/web/http/headers/accept-charset/index.md
Comment (jasonren0403) [inline files/zh-cn/web/http/headers/accept-charset/index.md:1]: Not sure if we have other occurences of 'old spec style' like this in zh-CN documents

Comment (yin1999) [reply files/zh-cn/web/http/headers/accept-charset/index.md:1]: I've done [the convertion](https://github.com/mdn/translated-content/pull/7455) while converting all html files to markdown. So there might be some occurences of this, but should not be too many, just remove when got one ;)

File: files/zh-cn/web/css/css_colors/index.md [hunk 1]
```diff
@@ -36,11 +36,10 @@ original_slug: Web/CSS/CSS_Color
...
-{{Compat}}
-
 ## 参见
 
-- 有与颜色有关属性的其他规范：{{cssxref("background-color")}}、{{cssxref("border-color")}}、{{cssxref("outline-color")}}、{{cssxref("text-decoration-color")}}、{{cssxref("text-emphasis-color")}}、{{cssxref("text-shadow")}}、{{cssxref("caret-color")}} 和 {{cssxref("column-rule-color")}}
-- 在 CSS 中，渐变不是颜色，而是[图像](/zh-CN/docs/Web/CSS/CSS_images)。
+- [CSS 颜色调整](/zh-CN/docs/Web/CSS/CSS_color_adjustment)模块及 {{cssxref("print-color-adjust")}} 属性。
+- 定义 CSS [`<gradient>`](/zh-CN/docs/Web/CSS/gradient) 图像类型的 [CSS 图像](/zh-CN/docs/Web/CSS/CSS_Images) 模块。
+- [`VideoColorSpace`](/zh-CN/docs/Web/API/VideoColorSpace) 接口
+- [`<feColorMatrix>`](/en-US/docs/Web/SVG/Element/feColorMatrix) SVG 元素
```
Comment (yin1999) [inline files/zh-cn/web/css/css_colors/index.md:44]:
```suggestion
- [`<feColorMatrix>`](/zh-CN/docs/Web/SVG/Element/feColorMatrix) SVG 元素
```

File: files/zh-cn/web/http/headers/accept-charset/index.md [hunk 2]
```diff
@@ -43,18 +43,9 @@ Accept-Charset: utf-8, iso-8859-1;q=0.5
...
-
-## 浏览器兼容性
-
-{{Compat}}
-
 ## 参见
 
 - HTTP [内容协商机制](/zh-CN/docs/Web/HTTP/Content_negotiation)
+- [不要再用 Accept-Charset 了](https://hsivonen.fi/accept-charset/)
 - 用来表示内容协商结果的消息头：{{HTTPHeader("Content-Type")}}
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/accept-charset/index.md:50]:
```suggestion
- 用来表示内容协商结果的标头：{{HTTPHeader("Content-Type")}}
```

File: files/zh-cn/web/http/headers/accept-charset/index.md [hunk 3]
```diff
@@ -43,18 +43,9 @@ Accept-Charset: utf-8, iso-8859-1;q=0.5
...
-- 其他类似消息头：{{HTTPHeader("TE")}}, {{HTTPHeader("Accept-Encoding")}}, {{HTTPHeader("Accept-Language")}}, {{HTTPHeader("Accept")}}
+- 其他类似消息头：{{HTTPHeader("TE")}}、{{HTTPHeader("Accept-Encoding")}}、{{HTTPHeader("Accept-Language")}}、{{HTTPHeader("Accept")}}
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/accept-charset/index.md:51]:
```suggestion
- 其他类似的标头：{{HTTPHeader("TE")}}、{{HTTPHeader("Accept-Encoding")}}、{{HTTPHeader("Accept-Language")}}、{{HTTPHeader("Accept")}}
```

**Review by yin1999 — APPROVED** [review]
Great, just some fixes

**Pull request merged**